src/main/java/es/urjc/cloudapps/library/application/BookService.java

```diff
@@ -1,13 +1,23 @@
 package es.urjc.cloudapps.library.application;
 
+import es.urjc.cloudapps.library.data.BookDto;
 import es.urjc.cloudapps.library.data.InMemoryBookRepository;
 import es.urjc.cloudapps.library.domain.Book;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
+import java.util.List;
 import java.util.UUID;
 
+@Service
 public class BookService {
 
-    private InMemoryBookRepository repository;
+    @Autowired
```

Comment:
Como ves utilizar la inyección por constructor? De esa forma en los test se pueden sustituir las dependencias sin magia:
```suggestion
public BookService(InMemoryBookRepository bookRepository) {
this.bookRepository = bookRepository;
 }
```